templates/common/.github/workflows/bicep/azure-dev.yml

```diff
@@ -19,8 +19,6 @@ permissions:
 jobs:
   build:
     runs-on: ubuntu-latest
-    container:
```

Comment:
By removing this, CI won't work because it won't have all of the dependencies needed.  Like .NET, Java, Node, etc.

Reply:
We should move to using more idomatic github action ways of installing the toolchain (just use `setup-dotnet`, `setup-node`, etc) to install just the tools you want.

Reply:
What if we use `@devcontainers/cli` to run CI/CD?  Just like how we do for azd?
That would allow us to use the `ghcr.io/devcontainers/features/` that are already defined for the template's devcontainer.
I see a constant pain on keeping/maintaining CI/CD + devcontainer independently...

Reply:
> We should move to using more idomatic github action ways of installing the toolchain (just use `setup-dotnet`, `setup-node`, etc) to install just the tools you want.

That's fine, just can't remove and not have something to replace it with

Reply:
> That's fine, just can't remove and not have something to replace it with

Yup. Agreed!

Reply:
@hemarina - The gist of this is that if we remove the container from this, we'll need to have actions to install all of the dependencies.  Or create a container for each.

Reply:
You'll likely need to set the appropriate versions, for example calling the dotnet set version action, so that the templates use the right versions of those dependencies.

Reply:
If you don't lock to a version, then when the GHA default container has a new one it could fail.  i.e., backwards compat issues with new version, etc.